spec/javascripts/components/story/expanded_story/expanded_story_notes_spec.js

```diff
@@ -51,47 +53,89 @@ describe('<ExpandedStoryNotes />', () => {
     expect(wrapper.find('NotesList')).toExist();
   });
 
-  it('disables the add note button if text area is empty', () => {
-    const story = newStory();
-
-    const wrapper = shallow(
-      <ExpandedStoryNotes
-        story={story}
-        onCreate={onCreate}
-        onDelete={onDelete}
-      />
-    );
-
-    const textArea = wrapper.find('.create-note-text');
-    const button = wrapper.find('.create-note-button input');
-
-    textArea.simulate('change', { target: { value: '' } });
-    expect(button.prop('disabled')).toBe(true);
-  });
+  describe('when component is enabled', () => {
+    describe('when user create a new note', () => {
+      it('triggers the onCreate callback passing the note', () => {
+        const story = newStory();
+  
+        const change = 'newNote';
+  
+        const onCreateSpy = sinon.spy();
+  
+        const wrapper = shallow(
+          <ExpandedStoryNotes
+            story={story}
+            onCreate={onCreateSpy}
+            onDelete={onDelete}
+            disabled={false}
+          />
+        );
+  
+        const textArea = wrapper.find('.create-note-text');
+        const button = wrapper.find('.create-note-button input');
+  
+        textArea.simulate('change', { target: { value: change } });
+        button.simulate('click');
+  
+        expect(onCreateSpy).toHaveBeenCalledWith(change);
+      });
+    });
 
-  describe('when user create a new note', () => {
-    it('triggers the onCreate callback passing the note', () => {
+    it('disables the add note button if text area is empty', () => {
       const story = newStory();
-
-      const change = 'newNote';
-
-      const onCreateSpy = sinon.spy();
-
+  
       const wrapper = shallow(
         <ExpandedStoryNotes
           story={story}
-          onCreate={onCreateSpy}
+          onCreate={onCreate}
           onDelete={onDelete}
+          disabled={false}
         />
       );
-
+  
       const textArea = wrapper.find('.create-note-text');
       const button = wrapper.find('.create-note-button input');
+  
+      textArea.simulate('change', { target: { value: '' } });
+      expect(button.prop('disabled')).toBe(true);
+    });
+  });
 
-      textArea.simulate('change', { target: { value: change } });
-      button.simulate('click');
+  describe('when component is disabled', () => {
+    it('does not render a form', () => {
+      const notes = [{id: 0, note: 'foo'}]
+      const story = newStory(notes);
+  
+      const wrapper = shallow(
+        <ExpandedStoryNotes
+          story={story}
+          onCreate={onCreate}
+          onDelete={onDelete}
+          disabled={true}
+        />
+      );
+      
+      const btnExsists = wrapper.exists('.create-note-button');
+      const formExists = wrapper.exists('.create-note-text');
+      expect(btnExsists && formExists).toBe(false);
```

Comment:
We should avoid `&&` in assertions like that. If we need to check the presence of two things, we should create to `it`.

Reply:
Got it.